easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.2.1-CrayGNU-17.08-cuda-8.0-python3.eb

```diff
@@ -33,6 +33,7 @@ dependencies = [
     ('SWIG', '3.0.10', ver_suffix_py),
     ('protobuf', '3.1.0', ver_suffix_py),
     ('backports.weakref', '1.0rc1', ver_suffix_py),
+    ('Werkzeug', '0.12.2', ver_suffix_py),
```

Comment:
eb file for this dep is missing?

Reply:
Hopefully, now all deps are commited...

Reply:
BTW: Can we also push the changes to Daint? The Python package Werkzeug is also required for TF 1.2.1. TF 1.3. should also be also stable and we might need it for some larger scaleability test.

Reply:
It's up to you to decide if this goes to daint or not, since you are in charge of supporting it.

As for the dependency `Werkzeug`, I'd suggest to add as a hidden dependency (since users are not asking to use that as standalone module without TF). You can updated the hide deps list [here](https://github.com/eth-cscs/production/blob/master/easybuild/easybuild.d/system_wide.cfg#L3)

If you want to update also TF 1.2.1, I suggest to open another PR (once this one is merged, in order to make the testing easier).

Reply:
PR testing didn't pass yet, so you can update TF 1.2.1 here as well, it won't make much difference.